jscomp/core/jsoo_main.ml

```diff
@@ -136,13 +136,13 @@ module Js = struct
   type js_string
   external string : string -> js_string t = "caml_js_from_string"
   external to_string : js_string t -> string = "caml_js_to_string"
-  external fs_register : js_string t -> js_string t -> unit = "caml_fs_register"
+  external create_file : js_string t -> js_string t -> unit = "caml_create_file"
```

Comment:
is this the only change that makes it will not work with 2.8.4 but 3.0?

Reply:
Yes, it's one of the 2 breaking changes: https://github.com/ocsigen/js_of_ocaml/commit/0ec170a1e99a2a46c77e27acec3819f23ee26d3c#diff-6e61909b542c36eb53e3df028cf42e83